src/objects/container/container.scss

```diff
@@ -0,0 +1,16 @@
+@use "../../design-tokens/layout.yml";
+
+.o-container__content {
```

Comment:
I notice you are only styling the `__content`. What is the outer container for?

Reply:
I imagine we will the ability to add margins to the container. Although the outer container doesn't have any styles, if we removed it and added space utilities to the `__content` it negates the margins that help to center the content.

Reply:
That's a fine explanation.

If I were a newcomer to the project, where would I find that explanation or direction for where to add margin/padding?

Reply:
Do you think that's something that should be added to the container description?

Reply:
I think it'd be good to mention in the story docs (the `.mdx` file) _and_ probably as a comment in this file to explain why we're _starting_ with `o-container__content`. That way it's clearly communicated to CSS developers _and_ to template designers!

Reply:
Sounds good thanks, I updated the story docs already, but I'll add an explanation to this file as well.